README.md

```diff
@@ -43,184 +42,128 @@ This solution has the EKS cluster deployed on AWS.
-
-```shell
 make integ-awscdk-deploy
-```
-
-Test the deploy AWS CDK solution.
-
-```shell
 make integ-awscdk-test
 ```
 
-Restart LocalStack inside the running Pod. Now you can deploy again and retest.
-
-```shell
-make reset-ls
-```
+After the test passes, let's cleanup the EKS cluster:
 
-**Cleanup EKS Cluster**
-make aws-cleanup-ns NS_NUM=<your namespace number>
 ```shell
-make aws-cleanup-ns NS_NUM=0
+make aws-deploy-cleanup NS_NUM=0
 make aws-cleanup-cluster
 ```
 
-
 ### Solution-2
 
 This solution has the EKS cluster deployed on your local machine, using the EKS anywhere plugin.
 
 #### Prerequisites for Solution-2 🧰
 
-- A laptop
+- Locally-accessible machine.
 - [install Helm](https://helm.sh/docs/intro/install/)
 - [install kubectl](https://kubernetes.io/docs/tasks/tools/)
 - [install eksctl](https://eksctl.io/installation/)
 - [install eksanywhere plugin](https://anywhere.eks.amazonaws.com/docs/getting-started/install/)
 
-#### Create EKS anywhere cluster
-
-Creating the cluster takes a couple minutes. Select a K8S Namespace number and supply it as an argument.
-```shell
-make eksany-create-cluster NS_NUM=0
-```
+#### Solution-2 Steps
 
-![EKSAClusterDeploy](./docs/eksa-deploy-cluster.png "EKSAClusterDeploy")
+Let's create the AWS cluster using EKS Anywhere locally. This blueprint builds namespaces in the format of `ls<NS_NUM>`. So, we're going to choose a namespace number for the following targets.
 
-Create Namespace and setup Coredns
 ```shell
-make eksany-setup-ns NS_NUM=0
-```
-
-#### Deploy LocalStack and DevPod
+make local-setup-cluster
 
-```shell
-make eksany-deploy-ls NS_NUM=0
-```
+# Create the namespace.
+make local-bootstrap NS_NUM=0
+# Apply CoreDNS patch so that CoreDNS points to the Localstack service.
+make patch-coredns NS_NUM=0
 
-#### Run Test
-Get a shell on the DevPod
-```shell
-make eksany-ssh-devpod NS_NUM=0
-```
+# Generate manifests and apply Localstack/DevPod deployments.
+make deploy-setup NS_NUM=0
+make deploy-localstack NS_NUM=0
 
-**Clone solution repo and test**
-```shell
-git clone https://github.com/localstack-samples/lambda-ddb.git
+# Exec into dev environment
+make exec-devpod-interactive NS_NUM=0
 ```
 
-Get into the repo dir.
+Once inside the DevPod environment, let's clone our Localstack sample project:
 
 ```shell
+git clone https://github.com/localstack-samples/lambda-ddb.git
 cd lambda-ddb
-```
 
-Bootstrap the solution. This is a solution built with AWS CDK.
-
-```shell
 make integ-awscdk-bootstrap
+make integ-awscdk-deploy
+make integ-awscdk-test
 ```
 
-Deploy the AWS CDK solution.
+After the test passes, let's cleanup the EKS cluster:
 
 ```shell
-make integ-awscdk-deploy
+make local-deploy-cleanup NS_NUM=0
+make local-cleanup-cluster
 ```
 
-Test the deploy AWS CDK solution.
+### Multiple Namespaces
 
-```shell
-make integ-awscdk-test
-```
+To deploy multiple Localstack instances with their own dev environment, you can do something like this:
 
-#### Setup Second Namespace
```

Comment:
Keep this section in the readme for easy reference to basic cmd line steps

Reply:
We added a `### Multiple Namespaces` section that creates `n` number of these environments. Wouldn't that be more comprehensive though? What do you say?

Reply:
On second note: we still want this for strictly the demo.

Reply:
Done!